### DISCUSSION TITLE
so-we-have-a-memory-leak

### DISCUSSION THREAD

**Discussion by giscus** [stevenharman/stevenharman.github.io#17] [Announcements]
# so-we-have-a-memory-leak

A story of finding, understanding, and fixing a memory leak in our Rails app. With graphs and code!

https://stevenharman.net/so-we-have-a-memory-leak

**Comment by stevenharman** [comment]
Apologies @jasonkarns, I swapped out the comment provider thus losing your (much appreciated) pedantry re: unnecessary `export`-ing in the shell examples. I'd been wanting and meaning to get rid of Disqus for a while, but only just today discovered [giscus](https://giscus.app).

**Reply by jasonkarns** [reply]
lol, that's actually a bit of a relief 😂
